code/modules/reagents/chemistry/machinery/chem_recipe_debug.dm

```diff
@@ -105,25 +153,207 @@
 				minorImpurity++
 			if(R2.volume < C.results[R])
 				impure_string += "Reaction [cached_reactions[index]] has a product [R] <span class='warning'>[R2.volume]u</span> purity of [R2.purity] index:[index]\n"
+		if(beaker_spawn && reagents.total_volume)
+			var/obj/item/reagent_containers/glass/beaker/bluespace/B = new /obj/item/reagent_containers/glass/beaker/bluespace(loc)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names